hunt/app/views/animals/show.html.erb

```diff
@@ -10,6 +10,13 @@
 
           <p class="card-text">State: <%=@animal.state%></p>
           <p class="card-text">Category: <%=@animal.category%></p>
+          <p class="card-text">Season:
+          <%= if @animal.season_open.present? && @animal.season_close.present?
+            @animal.season_open.strftime('%b %d %Y') + " - " + @animal.season_close.strftime('%b %d %Y')
```

Comment:
Same thing here, if one date is set it should still show. I think we'll want to move the "Season" label inside the condition also, so that we don't show it if neither date is set. This will probably be easier with that other syntax I mentioned.